india_compliance/gst_india/doctype/gstr_1_beta/gstr_1_export.py

```
@@ -419,10 +462,12 @@
                 "fieldname": inv_f.DOC_DATE,
                 "data_format": {"number_format": self.DATE_FORMAT},
                 "header_format": {"width": ExcelWidth.XS.value},
+                "transform": lambda x: x.strftime("%d-%b-%y"),
             },
             {
-                "label": _(gov_xl.NOTE_TYPE),
-                "fieldname": inv_f.TRANSACTION_TYPE,
+                "label": _(gov_xl.NOTE_TYPE.value),
```

Pylint (3.3.7)

[error] Instance of 'str' has no 'value' member

(E1101)